zb-db/src/main/java/io/zeebe/db/impl/rocksdb/transaction/ZeebeTransactionDb.java

```diff
@@ -343,6 +301,7 @@ private void foreach(
    * <p>While iterating over subsequent keys we have to validate it.
    */
   protected <KeyType extends DbKey, ValueType extends DbValue> void whileEqualPrefix(
+      final DbLong columnFamilyKey,
```

Comment:
Where do we actually test this? I can't find it.

Reply:
Not sure what you mean @deepthidevaki The functionality of `whileEqualPrefix`? Can you elaborate on that?

Reply:
API wise nothing has changed. We have the same tests, since they expect usage of multiple column families. We can put and get values from different column families and also iterate over them. This works and is tested in zb-db. We have tests with one but also with multiple column families. Plus in the engine we have tests where we use 50 column families.